foodsaving/history/tests/test_api.py

```diff
@@ -295,6 +295,6 @@ def test_filter_by_date(self):
         self.client.force_login(self.member)
         self.client.post('/api/groups/', {'name': 'xyzabc', 'timezone': 'Europe/Berlin'})
         response = self.get_results(history_url, data={'date_1': timezone.now()})
-        self.assertEqual(len(response.data), 1)
+        self.assertEqual(len(response.data), 2)
```

Comment:
Does this create a history event because the group creator joined the group?

Reply:
yep